code/modules/atmospherics/machinery/portable/scrubber.dm

```diff
@@ -1,5 +1,6 @@
 /obj/machinery/portable_atmospherics/scrubber
 	name = "portable air scrubber"
+	desc = "A small portable scrubber, which removes bad gasses from the surrounding air."
```

Comment:
It can remove good gasses from the surrounding air too

Reply:
well yea but the "intended" way nanotrasen wants you to use it is for bad gasses

Reply:
just have it say "Gasses"